File: Curso Logica de Programação Neri Neitzke/Algoritmos em Java/vendas2.java
// Prof Neri Aldoir Neitzke
// www.INFORMATICON.com.br - videoaulasneri@gmail.com
/* Problema.: Uma Empresa de desenvolvimento de softwares paga a seu vendedor
um fixo de R$ 500,00 por mês, mais um bônus de R$ 50,00 por sistema vendido.
Faça uma algoritmo que leia quantos softwares o funcionário vendeu e determine
 o salário total do funcionário. Mostre as informações que você achar necessário.
*/
import javax.swing.JOptionPane;
class vendas2
{
   public static void main(String args[])
   {
       float fixo, bonus, qtd_softw_vend,  salario_total ;
       fixo = 500;
       qtd_softw_vend = Float.parseFloat(JOptionPane.showInputDialog("Digite a quantidade de software vendido "));
       bonus = qtd_softw_vend * 50;
       salario_total = fixo + bonus;
       JOptionPane.showMessageDialog(null,"O salario Fixo = "+fixo+
                                          "\nO bonus  = " + bonus+
                                          "\nO salario total = "+salario_total);
   }
}

File: Curso Logica de Programação Neri Neitzke/Algoritmos em Pascal/pascal/vendas2.pas
// Prof Neri Aldoir Neitzke
// www.INFORMATICON.com.br - videoaulasneri@gmail.com
{2.	Problema.: Uma Empresa de desenvolvimento de softwares paga a seu vendedor
 um fixo de R$ 500,00 por mês, mais um bônus de R$ 50,00 por sistema vendido.
 Faça uma algoritmo que leia quantos softwares o funcionário vendeu e determine
 o salário total do funcionário. Mostre as informações que você achar necessário.}
program vendas2;
var fixo, bonus, qtd_softw_vend,  salario_total : Real;
begin
      fixo := 500;
      write('Digite a quantiade de softwares vendidos.: ');
      readln(qtd_softw_vend);
      bonus := qtd_softw_vend * 50;
      Salario_total := fixo + bonus;
      writeln('O salário fixo = ',fixo:6:2);
      writeln('o valor de bonus = ',bonus:6:2);
      writeln('o salario total  = ',salario_total:6:2);
      writeln('..................Final do Sistema..................... ');
      readln();
end.
